### PR TITLE
Fix blurhash

### PR DESCRIPTION
**Pull request by Meldiron**

### CODE REVIEW

File: src/lib/blurhash.json
Comment (ItzNotABug) [inline src/lib/blurhash.json:1]: Why a different file and not in the same `products` object?

**Comment by ItzNotABug** [comment]
Why a different module and not just add it in the core project? For usability and separation of concern?

**Review by ItzNotABug — APPROVED** [review]
A few comments, rest looks good.

File: src/lib/blurhash.json
Comment (Meldiron) [reply src/lib/blurhash.json:1]: It's auto-generated by CI/CD, so better keep it in separate file. I would be worried for CI/CD to affect entire store

Comment (ItzNotABug) [reply src/lib/blurhash.json:1]: I doubt things could go wrong at this point now but this works too.

**Pull request merged**